package.json

```diff
@@ -12,10 +12,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com:Zextras/carbonio-ui-sdk.git"
+    "url": "git+https://github.com/zextras/carbonio-ui-sdk.git"
   },
   "engines": {
-    "node": "v18",
+    "node": "v20",
```

Comment:
By replacing it, we are removing support for node 18. We could keep both node 18 and node 20